src/esp/gfx/CMakeLists.txt

```diff
@@ -122,9 +118,6 @@ add_library(
 )
 
 if(BUILD_WITH_CUDA)
-  # We currently don't directly depend on gfx_batch for anything, just taking
-  # the header-only CUDA helpers from there, so gfx_batch isn't in
-  # target_link_libraries().
   target_include_directories(
```

Comment:
You could omit this now altogether, by changing the `PRIVATE` to `PUBLIC` in gfx_batch/CMakeLists.txt.